maven-embedder/src/main/java/org/apache/maven/cli/transfer/ConsoleMavenTransferListener.java

```diff
@@ -35,7 +35,7 @@
  */
 public class ConsoleMavenTransferListener extends AbstractMavenTransferListener {
 
-    private Map<TransferResource, Long> transfers = new LinkedHashMap<>();
+    private Map<TransferResourceIdentifier, TransferResourceAndSize> transfers = new LinkedHashMap<>();
     private FileSizeFormat format = new FileSizeFormat(Locale.ENGLISH); // use in a synchronized fashion
```

Comment:
We need to get rid of the `Locale.ENGLISH` parameter which is unused, though this can be done in a subsequent PR.

Reply:
Addressed in https://github.com/apache/maven/pull/1579